### DISCUSSION TITLE
Cannot find module 'think-model-mysql'

### DISCUSSION THREAD

**Discussion by lizheming** [vercel/vercel#8494] [Help] [answered]
I have deployed a new app https://waline-test-test.vercel.app/ , it return error give me some tips about 'cant\'t find moudle'. But the module has declared in my dependencies. 

https://github.com/lizheming/waline-test-test

![image](https://user-images.githubusercontent.com/424491/187733729-82d1b01a-6c79-42f4-b36f-388f39faed64.png)

my project depdens module `@waline/vercel` and the missing module `think-model-mysql` has declared in its deps.
![image](https://user-images.githubusercontent.com/424491/187734001-eba4362e-af4c-4a7b-b64c-4c128d2c918f.png)

**Comment by amyegan** [comment, marked as answer]
I suspect this is happening because of the deprecated [builds](https://vercel.com/docs/project-configuration#legacy/builds) property in your `vercel.json` excluding unspecified files. 
> Note: When at least one builds item is specified, only the outputs of the build processes will be included in the resulting deployment as a security precaution. This is why we need to allowlist static files explicitly with@vercel/static.

You should consider switching to the [functions](https://vercel.com/docs/project-configuration#project-configuration/functions) property. Or you could move your serverless functions into an `/api` directory at the root of your Project with no other configuration needed.

**Reply by lizheming** [reply]
Seems there has some bug with `@vercel/node@2.5.11` which is the latest version. It runs well when I rollback to `@vercel/node@2.5.10`。

**Reply by amyegan** [reply]
Thanks for sharing this @lizheming. There was a `@vercel/node` update yesterday to [v2.5.13](https://www.npmjs.com/package/@vercel/node/v/2.5.13). Is this problem still happening for you on this new version?

**Reply by lizheming** [reply]
Yes, I have tried `@vercel/node@2.5.12` and `@vercel/node@2.5.13`, both of them can reproduce the problem.

**Reply by amyegan** [reply]
Thanks for trying that. I opened a new issue to have the team look into the source of the problem https://github.com/vercel/vercel/issues/8513

**Reply by amyegan** [reply]
Hello again @lizheming! Wanted to share that we have an update on the issue, in case you hadn't already seen it.

https://github.com/vercel/vercel/issues/8513#issuecomment-1238604472
